package.json

```diff
@@ -1,11 +1,16 @@
 {
   "name": "ozone-e2e",
-  "version": "1.0.0",
+  "version": "1.0.0-SNAPSHOT",
   "description": "",
   "main": "index.js",
+  "files": [ "**/*", "!node_modules/","!playwright-report/", "!test-results/", "!**/.git/" ],
```

Comment:
@kdaud, have you confirmed that this will acatually ignore .github/.gitignore folder/file? Shouldn't it look some thing like 

`  "files": [ "**/*", "!node_modules/","!playwright-report/", "!test-results/", "!**/.git*/" ],`

Reply:
Sure, I have tested and the .git/ files were being  included among the published files. Thanks @Ruhanga